inspirehep/modules/workflows/workflows/article.py

```diff
@@ -390,45 +383,23 @@
         ),
     ),
 
-    IF_ELSE(
-        is_marked('already-in-holding-pen'),
-        IF_ELSE(
-            has_same_source('holdingpen_matches'),
-            # stop the matched wf and continue this one
-            [
-                stop_matched_holdingpen_wfs,
-                mark('stopped-matched-holdingpen-wf', True),
```

Comment:
I don't think this logic should disappear: if there are two updates that need to be processed, it's better to skip the earliest one, otherwise the curator has to work twice.

Reply:
No, as I understand from what Antonio explained me, if curator will accept/reject it once Inspire will remember what to do for all other workflows matched by this one. If not, then this code really has to be restored.

Reply:
@pazembrz that is true only if we are talking about Rejected workflows: if a curator rejects a paper, the next update for that paper will be auto-rejected. Instead, when a curator edits the metadata of the first workflow and then a new update comes (with the same incorrect data), probably the curator should correct again the same data.
That's why we should keep the logic of 'stop the current and process the newest'.